content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,30 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Documentation content standards
+
+Kubernetes documentation does not host certain types of content:
```

Comment:
Rephrase this to use fewer bullets and [simpler, more direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language).

Also, we can clarify that we host content from *current* CNCF projects.
```
Kubernetes documentation only hosts content from projects within the `kubernetes` or `kubernetes-sigs` GitHub organizations, or from current CNCF projects.

Kubernetes documentation does not host content identical content sourced from multiple locations ("dual-sourced" content). 
```

Reply:
updated